app/assets/javascripts/backbone/views/form_items_view.js.coffee

```diff
@@ -0,0 +1,93 @@
+class ELMO.Views.FormItemsView extends Backbone.View
+
+  el: '.form-items'
+
+  events:
+    'click .add-group': 'show_new_group_modal'
+    'click .form-group .edit': 'show_edit_group_modal'
+    'click .form-group .delete': 'delete_group'
+
+  initialize: (params) ->
+    this.nested_list()
+    this.params = params
+
+  show_new_group_modal: (link) ->
```

Comment:
`link` is not really an appropriate name. The argument is an event object. Convention is to just call it `event`.